lms/urls.py

```diff
@@ -267,6 +267,11 @@
             'courseware.module_render.xblock_resource',
             name='xblock_resource_url'),
 
+        # xblock Rendering
+        url(r'^xblock/{usage_key_string}$'.format(usage_key_string=settings.USAGE_KEY_PATTERN),
```

Comment:
@andy-armstrong Thanks for raising this.  I will add a comment to make this clear.